app/views/hiring_staff/vacancies/copy/new.html.haml

```diff
@@ -0,0 +1,51 @@
+- content_for :page_title_prefix, "#{@copy_form.errors.present? ? 'Error: ' : ''}Copy a job for #{@school.name}"
+
+%h1.govuk-heading-l
+  = t('jobs.copy_page_title', job_title: @copy_form.vacancy.job_title.downcase)
+  %span.govuk-caption-l
+    Step 1 of 2
```

Comment:
The existing review page does not have a step count. It is used in this copy journey which has 2 steps, and the original publishing journey which has 3 steps - so it wouldn't work if it did!

I do however think there's merit in having this for the user so that they trust that there will be a second step where they can do more, the copy is *not* limited only to the fields we're showing them.